demos/MoAE/moae_fmriprep.m

```diff
@@ -0,0 +1,118 @@
+% This script will run the FFX and contrasts on it of the MoAE dataset
+% using the fmriprep preprocessed data
+%
+% If you want to get the preprocessed data and you have datalad on your computer
+% you can run the following commands to get the necesary data::
+%
+%
+%   datalad install --source git@gin.g-node.org:/SPM_datasets/spm_moae_fmriprep.git \
+%           inputs/fmriprep
+%   cd inputs/fmriprep && datalad get *.json \
+%                     */*/*tsv \
+%                     */*/*json \
+%                     */*/*desc-preproc*.nii.gz \
+%                     */*/*desc-brain*.nii.gz
```

Comment:
main file to look at

https://github.com/cpp-lln-lab/CPP_SPM/blob/f1b0cfe7ba7bc1b315e6cfe8e7aeefecbea66773/demos/MoAE/moae_fmriprep.m

Reply:
@mwmaclean this is the file you want to have a look at